assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -134,6 +134,9 @@ const toggleSortLookup = {
   [SORT_DIRECTION.ASC]: SORT_DIRECTION.DESC,
 };
 
+const titleFormatted = (rawTitle) => {
+  return rawTitle === '' ? __('(no title)', 'web-stories') : rawTitle;
```

Comment:
Should we do `rawTitle.trim() === ''` just to be extra sure for titles with empty spaces?